modules/nw-example-configs-for-externalip.adoc

```diff
@@ -0,0 +1,8 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-example-configs-for-externalip_{context}"]
```

Comment:
Considering pulling all of the example configs into one reference module.

Reply:
Most `nw-` related modules are of the form `nw-[subject]`, so maybe `nw-externalip-example-configs.adoc` for consistency?